components/feature/media/src/main/java/mozilla/components/feature/media/focus/AudioFocus.kt

```diff
@@ -23,19 +20,34 @@ import mozilla.components.support.base.log.logger.Logger
  * https://developer.android.com/guide/topics/media-apps/audio-focus
  */
 internal class AudioFocus(
-    private val context: Context
+    val audioManager: AudioManager
 ) : AudioManager.OnAudioFocusChangeListener {
     private val logger = Logger("AudioFocus")
     private var playDelayed = false
     private var resumeOnFocusGain = false
 
+    private val audioFocusController = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+        AudioFocusControllerV26(audioManager, this)
```

Comment:
Does worth creating a `AudioFocusControllerCompact` class to hide V26 and V21 and avoid versions comparisons?

Reply:
I'm not sure how this would help? 🤔 Those are also all internal classes!

Reply:
Ok, I thought that you will continue using it in other places. If it just here doesn't worth it 👍